### PR TITLE
Pass chatbot latest prompt

### LINKED ISSUE #245: 🐛 Bug Report: HTTP 500 when clicking on send button of chatbot
### 📜 Description

 HTTP 500 when clicking on send button of chatbot

### 👟 Reproduction steps

Go to chatbot -> Type any prompt -> Click on send

### 👍 Expected behavior

No error

### 👎 Actual Behavior with Screenshots

c1

### 💻 Operating system

Windows

### What browsers are you seeing the problem on?

Chrome

### 📃 Provide any additional context for the Bug.

_No response_

### 📷 Relevant Screenshots (Links)

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🔗 Are you willing to submit PR?

Yes, I am willing to submit a PR!

### 🧑‍⚖️ Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mehul-m-prajapati**
### Related Issue
Closes #245 

---

### Description
- Pass query string to chatbot
---

### How Has This Been Tested?
Locally
---

### Screenshots (if applicable)

---

### Type of Change

- [x] Bug fix
- [ ] New feature
- [ ] Code style update
- [ ] Breaking change
- [ ] Documentation update

### CODE REVIEW

**Comment by kom-senapati** [comment]
Its not query, add the initial prompt

File: app/api_routes.py [hunk 1]
```diff
@@ -309,7 +309,7 @@ def api_chatbot(chatbot_id: int) -> Union[Response, tuple[Response, int]]:
     query: str = data.get("query")
     apikey = request.headers["apikey"]
     engine = request.headers["engine"]
-    chat_to_pass: List[Dict[str, str]] = [{"role": "system", "content": chatbot.prompt}]
+    chat_to_pass: List[Dict[str, str]] = [{"role": "system", "content": query}]
```
Comment (priyanshuverma-dev) [inline app/api_routes.py:312]: We need this

```suggestion
    chat_to_pass: List[Dict[str, str]] = [{"role": "system", "content": chatbot.latest_version.prompt}]
```

**Review by priyanshuverma-dev — CHANGES_REQUESTED** [review]
Please see the changes that are required

File: app/api_routes.py (hunk 1 above)
Comment (mehul-m-prajapati) [reply app/api_routes.py]: @priyanshuverma-dev : thanks for suggestion

**Comment by kom-senapati** [comment]
LGTM

**Pull request merged**